tests/api/circulation/test_loan_extend.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019-2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test loan extend."""
+
+from invenio_circulation.api import Loan
+from invenio_circulation.proxies import current_circulation
+
+from ..helpers import user_login
+
+
+def test_loan_extend_stuff(
+    app, client, json_headers, users, testdata, app_config, loan_params
+):
+    """Test loan can be extended."""
+    # Create a Loan with patron_pid: 1
+    user_login(client, "admin", users)
+    loan_data = testdata["loans"][1]
+    loan = Loan.get_record_by_pid(loan_data["pid"])
+
+    # FIXME: AttributeError: '_AppCtxGlobals' object has no attribute 'identity'
```

Comment:
#HELPME! I might be doing it all wrong.

Reply:
try to use `from flask_security import login_user` instead